api/fetch-pokemons.js

```diff
@@ -3,15 +3,15 @@ import { fetchData } from '@/utils/fetch-data';
 export const fetchPokemons = async ({ limit = 10, offset = 0 } = {}) => {
   try {
     const data = await fetchData({
-      URL: `https://pokeapi.co/api/v2/pokemon`,
+      URL: process.env.NEXT_PUBLIC_POKEMON_URL,
       queryParams: { limit, offset },
     });
 
     const pokemonDetails = [];
 
     let id = offset;
 
-    const image = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/`;
+    const image = process.env.NEXT_PUBLIC_POKEMON_IMAGE;
```

Comment:
You don't need to save this in the environment file, save it in the constants file and import here.